src/nav/infrastructure.yml

```diff
@@ -176,6 +176,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration
           - title: Collectd integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/collectd-integration
+          - title: dbt Cloud with Snowflake and Airflow Integration 
```

Comment:
This section is for integrations that use the infrastructure agent, but dbt Cloud is not using the infrastructure agent. Are we sure it belongs here?